metadata.yaml

```
@@ -0,0 +1,8 @@
+name: sssd-operator
```

Should the name of the charm be "sssd-client"?

```suggestion
name: sssd-client
```

---

I was going to make this change but I double-checked https://juju.is/docs/sdk/naming#heading--naming-charms and it seems the suggestion is to avoid suffixes like "x-client" or "x-server"; see drupal example. Following that I think sssd will work for the name

---

Just "sssd" works for me!